partcad/src/partcad/globals.py

```
@@ -97,16 +98,26 @@ def render(format=None, output_dir=None):
         template_name = "init-public.yaml"
     src_path = os.path.join(os.path.dirname(__file__), "template", template_name)
 
+    with open(src_path) as f:
+        config = yaml.load(f)
+    for key, value in config_options.items():
+        key = ''.join(x.capitalize() if i != 0 else x for i, x in enumerate(key.split('_')))
+        if value and value != "empty":
+            config.insert(0, key, value)
+
     if os.path.exists(dst_path):
         pc_logging.error("File already exists: %s" % dst_path)
```

Ruff (0.8.2)

Use format specifiers instead of percent format

Replace with format specifiers

(UP031)